python-checks/src/test/resources/checks/hotspots/unsafeHttpMethods/flask/views.py

```diff
@@ -8,6 +9,21 @@
 def sensitive1():
     return Response("Method: " + request.method, 200)
 
+# 127.0.0.1:5000/methods/sensitive1
+@other_routes.route('/sensitive1', methods=['GET', 'POST'])  # Noncompliant
+def sensitive1_other_routes():
+    return Response("Method: " + request.method, 200)
+
+# 127.0.0.1:5000/methods/sensitive1
+@methods.route('/sensitive1', methods=['GET', 'POST'])  # Noncompliant
+def sensitive1():
+    return Response("Method: " + request.method, 200)
+
+# 127.0.0.1:5000/methods/sensitive1
+@other_routes.route('/sensitive1', methods=['GET', 'POST'])  # Noncompliant
+def sensitive1_other_routes():
+    return Response("Method: " + request.method, 200)
+
```

Comment:
I looks like the first and last tests are duplicate of each other, and the second one duplicates an existing test, and may not be valuable in the current context.